krm-functions/lib/ipallocation/v1alpha1/ipallocation_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+  Copyright 2023 Nephio.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/nokia/k8s-ipam/apis/ipam/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestParseKubeObjectNonEmpty(t *testing.T) {
```

Comment:
WOuld be a good idea to combine `TestParseKubeObjectNonEmpty` and `TestParseKubeObjectEmpty` into one single test method `TestParseKubeObject` and include both the test cases

Reply:
yes will do if we refactor this.